bin/ddtracerb

```diff
@@ -1,15 +1,3 @@
 #!/usr/bin/env ruby
-require 'datadog/profiling/tasks/exec'
-require 'datadog/profiling/tasks/help'
-
-command = ARGV.shift
-
-case command
-when 'exec'
-  Datadog::Profiling::Tasks::Exec.new(ARGV).run
-when 'help', '--help'
-  Datadog::Profiling::Tasks::Help.new.run
-else
-  puts "Command '#{command}' is not valid for ddtrace."
-  Datadog::Profiling::Tasks::Help.new.run
-end
+Kernel.warn 'WARNING: Use of `ddtracerb` is deprecated, and will be removed in 2.0. Use `ddprofrb` instead.'
+require_relative './ddprofrb'
```

Comment:
Bundler seems to get confused by this `require_relative`, since these `bin/` files get installed separately:

```
$ docker-compose run --no-deps --rm tracer-2.7 /bin/bash
root@8bf5c40760ee:/app# bundle install
# ...
root@8bf5c40760ee:/app# bundle exec ddtracerb exec ls
WARNING: Use of `ddtracerb` is deprecated, and will be removed in 2.0. Use `ddprofrb` instead.
bundler: failed to load command: ddtracerb (/usr/local/bundle/bin/ddtracerb)
Traceback (most recent call last):
	18: from /usr/local/bundle/bin/bundle:25:in `<main>'
	17: from /usr/local/bundle/bin/bundle:25:in `load'
	16: from /usr/local/bundle/gems/bundler-2.3.26/exe/bundle:36:in `<top (required)>'
	15: from /usr/local/bundle/gems/bundler-2.3.26/lib/bundler/friendly_errors.rb:120:in `with_friendly_errors'
	14: from /usr/local/bundle/gems/bundler-2.3.26/exe/bundle:48:in `block in <top (required)>'
	13: from /usr/local/bundle/gems/bundler-2.3.26/lib/bundler/cli.rb:25:in `start'
	12: from /usr/local/bundle/gems/bundler-2.3.26/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
	11: from /usr/local/bundle/gems/bundler-2.3.26/lib/bundler/cli.rb:31:in `dispatch'
	10: from /usr/local/bundle/gems/bundler-2.3.26/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
	 9: from /usr/local/bundle/gems/bundler-2.3.26/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	 8: from /usr/local/bundle/gems/bundler-2.3.26/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
	 7: from /usr/local/bundle/gems/bundler-2.3.26/lib/bundler/cli.rb:486:in `exec'
	 6: from /usr/local/bundle/gems/bundler-2.3.26/lib/bundler/cli/exec.rb:23:in `run'
	 5: from /usr/local/bundle/gems/bundler-2.3.26/lib/bundler/cli/exec.rb:58:in `kernel_load'
	 4: from /usr/local/bundle/gems/bundler-2.3.26/lib/bundler/cli/exec.rb:58:in `load'
	 3: from /usr/local/bundle/bin/ddtracerb:25:in `<top (required)>'
	 2: from /usr/local/bundle/bin/ddtracerb:25:in `load'
	 1: from /app/bin/ddtracerb:3:in `<top (required)>'
/app/bin/ddtracerb:3:in `require_relative': cannot load such file -- /app/bin/ddprofrb (LoadError)
```

I'm not sure it's worth spending time fixing this, perhaps faster to remove the `require_relative` and leave `ddtracerb` as a copy-pasted version of `ddprofrb` until we can delete it.

Reply:
Yeah that's what I originally did too; probably will revert this. Not worth the trouble.